iTunesSearch/iTunesSearch/Model Controller/SearchResultController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  SearchResultController.swift
+//  iTunesSearch
+//
+//  Created by Norlan Tibanear on 7/9/20.
+//  Copyright © 2020 Norlan Tibanear. All rights reserved.
+//
+
+import Foundation
+
+class SearchResultController {
+    
+    var searchResults: [SearchResult] = []
+    let baseURL = URL(string: "https://itunes.apple.com/search")!
+    
+    func performSearch(for searchTerm: String, resultType: ResultType, completion: @escaping () -> Void) {
```

Comment:
![Screen Shot 2020-07-13 at 10 06 28 PM](https://user-images.githubusercontent.com/23512386/87372294-2ca31a00-c555-11ea-9951-00103c472900.png)

You should have an optional Error? in this completion.